noaa/challenges/default.json

```diff
@@ -146,4 +146,123 @@
           "clients": 1
         }
       ]
+    },
+    {
+      "name": "top_metrics",
+      "description": "Compares the performance of top_metrics and top_hits",
+      "default": false,
+      "schedule": [
```

Comment:
As this is a new challenge, for this to be useful as a standalone unit there should be some ingested data as well, before executing the queries.

I guess you've been using it exclusively with `--pipeline=benchmark-only` against a pre-provisioned cluster with the indices present already, but I think we should have the standard `delete-index`, `create-index`, ..., `index` operations.

Reply:
Is there a good way for me to skip those when I'm running it as a test?

Reply:
But, yes, I did leave the ingest steps out on purpose because I end up doing so much iteration. I certainly can add them if you'd like.

Reply:
> Is there a good way for me to skip those when I'm running it as a test?

Yes you can use either the [include-tasks](https://esrally.readthedocs.io/en/stable/command_line_reference.html#include-tasks) or [exclude-tasks](https://esrally.readthedocs.io/en/stable/command_line_reference.html#exclude-tasks) cli flag.

Reply:
> But, yes, I did leave the ingest steps out on purpose because I end up doing so much iteration. I certainly can add them if you'd like.

Every challenge on the standard tracks should be runnable by default, so it'd be great if you can add them.